charts/fybrik/templates/fybrik-config.yaml

```diff
@@ -11,9 +11,9 @@ data:
   DATAPATH_MAX_SIZE: {{ .Values.manager.dataPathMaxSize | quote }}
   CONNECTION_TIMEOUT: {{ .Values.manager.connectionTimeout | default .Values.global.connectionTimeout | quote }}
   CATALOG_PROVIDER_NAME: {{ .Values.coordinator.catalog | quote }}
-  CATALOG_CONNECTOR_URL: {{ .Values.coordinator.catalogConnectorURL | default (printf "%s-connector:80" .Values.coordinator.catalog) | quote }}
+  CATALOG_CONNECTOR_URL: {{ .Values.coordinator.catalogConnectorURL | default (printf "http://%s-connector:80" .Values.coordinator.catalog) | quote }}
```

Comment:
There is another one in `opa-connector-cm.yaml`

Reply:
and probably other places where CATALOG_CONNECTOR_URL and/or MAIN_POLICY_MANAGER_CONNECTOR_URL are used in tests.

Reply:
Thanks, I changed that